docs/source/examples/Revisiting Lamberts problem in Python.md

```diff
@@ -78,7 +78,7 @@ ax.set_ylabel("$T$");
 
 ## Part 2: Locating $T_{min}$
 
-```python tags=["nbsphinx-thumbnail"]
+```python 
```

Comment:
Since you removed the tag, the thumbnail for the `Revisiting Lamberts problem...` example in the `Gallery` is no longer displayed. I do not think this is the reason behind documentation failures. Check previous behavior in associated [temporal documentation image](https://poliastro--1067.org.readthedocs.build/en/1067/gallery.html).

Reply:
Then why did the build pass when I removed the same tag? I am not able to understand where the problem is :thinking:

Reply:
Why is there a [semicolon](https://github.com/poliastro/poliastro/pull/1067/files#diff-386901cb412dd1888923f4a02f067485876ed36374c0fea15563f232a58565c4L76) here?

Reply:
> Then why did the build pass when I removed the same tag?

https://github.com/readthedocs/readthedocs.org/issues/7815

We will know next week.